tools/tdbg/tdbg_commands.go

```diff
@@ -463,98 +467,67 @@ func newAdminTaskQueueCommands() []*cli.Command {
 				},
 			},
 			Action: func(c *cli.Context) error {
-				return AdminListTaskQueueTasks(c)
+				return AdminListTaskQueueTasks(c, clientFactory)
 			},
 		},
 	}
 }
 
-func newAdminDLQCommands() []*cli.Command {
+func newAdminDLQCommands(clientFactory ClientFactory) []*cli.Command {
+	commonFlags := []cli.Flag{
+		&cli.StringFlag{
+			Name:  FlagDLQType,
+			Usage: fmt.Sprintf("Type of DLQ to manage. (options: namespace, history)"),
+		},
+		&cli.StringFlag{
+			Name:  FlagCluster,
+			Usage: "Source cluster",
+		},
+	}
```

Comment:
looks like shardID flag is missing here. That's still needed by the v1 dlq impl.